documentation/book/ref-minimum-resource-requirements.adoc

```diff
@@ -0,0 +1,16 @@
+[id='minimum-resource-requirements-{context}']
+= Minimum Resource Requirements
+
+[id='cluster-operator-resource-requirements-{context}']
+== Cluster Operator resource requirements
+
+Testing has shown that the Cluster Operator functions adequately with 256Mi of memory and 200m CPU when watching two clusters.
+It is therefore recommended to use these as a minimum when configuring resource requests and not to run it with lower limits than these.
+Configuring more generous limits is recommended, especially when it is controlling multiple clusters.
+
+[id='topic-operator-resource-requirements-{context}']
+== Topic Operator resource requirements
+
+Testing has shown that the topic operator functions adequately with 96Mi of memory and 100m CPU when watching two topics.
```

Comment:
We should not put into the documentation something what suggest we tested it only with 2 topics! I would remove this sentence. I think we should be also careful about the resources we put here. This is not fight for smallest possible CPU and Memory consumption. We should prefer good user experience over being cheap ... especially since a good Kafka cluster will consume siiignificantly more resources anyway and make it negligible whether the TO or CO needs 96Mi or 1Gi.

Reply:
I agree with you, but in this PR I was just trying to move around the content we _already had_. Are we going to make any suggestion about resources (for any pods) at this time?

Reply:
Well ... we might not get back to it to improve it before beta. So IMHO the _when watching two topics_ thing has to go away.